.github/workflows/tag.yaml

```diff
@@ -0,0 +1,37 @@
+name: Speakeasy Registry Tagger
+
+on:
```

Comment:
How will we trigger this on merge to main?

Reply:
Just a standard GH action trigger right?